bi-connectors/PowerBIConnector/OpenSearchProject-ODBC.mproj

```diff
@@ -36,7 +36,7 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="OpenSearch.pq">
+    <Compile Include="OpenSearchProject-ODBC.pq">
```

Comment:
Remove -ODBC

Reply:
Addressed in https://github.com/Bit-Quill/opensearch-project-sql/pull/27/commits/f27402a988e667b46f62712520cd5337128a3008